ilc/client/GuardManager.js

```diff
@@ -0,0 +1,32 @@
+import actionTypes from '../common/guard/actionTypes';
+
+export default class GuardManager {
+    #router;
+    #transitionHooksPlugin;
+
+    constructor(router, transitionHooksPlugin) {
```

Comment:
I would pass plugin manager instead of transitionHooksPlugin here

Reply:
Updated